docs/contributor-docs/release.md

```diff
@@ -78,15 +78,11 @@ git push origin release
 
 - The release process may take some time, be patient.
 
-##### 10. Announce the Release on Slack
+##### 10. Add a New Release to GitHub
 
-    - Once the release is successful, announce it on the #instui channel on Slack. Be sure to include the changelog and new version in your announcement.
+- Add a new release to GitHub. This will display the newly released version as the latest and automatically triggers a release notification in the #instui Slack channel.
```

Comment:
Should we talk about internal stuff i.e.: slack in the docs app?

Reply:
I would mention the automatic slack notification but maybe not the channel specifically